### DISCUSSION TITLE
da sonoff a zigbee

### DISCUSSION THREAD

**Discussion by truzzman** [indomus/forum#1952] [HUB: Home Assistant]
Buongiorno, attualmente ho un pò di sonoff con tasmota in giro per casa per varie funzioni (luci tapparelle irrigazione ecc) utilizzando una rete wifi 2.4 ad hoc solo per la parte domotica

Tra poco cambio casa e si tratta di una casa con muri in pietra spessi, il segnale wifi da alcune prove fatte non passa o solo con gran fatica.
Mi risulta che zigbee riesca a creare una mesh sostanzialmente migliorando la diffusione del segnale, cosa che mi tornerebbe molto comoda. Non ho mai usato zigbee ma non penso sia qualcosa di impossibile. 
Vorrei qualche consiglio per la spesa, su aliexpress ci sono prodotti Tuya zigbee Aubess a prezzi interessanti(10 meno di 70 €), cosa altro mi serve? l'hub giusto? e cosa consigliate?

Grazie per le vostre risposte e consigli.

Stefano

**Comment by drako92** [comment]
Ciao,
quindi se ho capito bene vorresti sostituire i tuoi attuali relè (attuatori) wifi con dei modelli zigbee ed integrarli in HomeAssistant?
In questo caso ti serve un gateway zigbee usb (ad esempio io uso ConBee II, ma ce ne sono di più economici) e poi devi decidere quale integrazione utilizzare in HomeAssistant tra Zigbee2MQTT, ZHA o deCONZ.

Io personalmente utilizzo il protocollo zigbee solo per i sensori di movimento, temperatura/umidità e apertura porte/finestre. Mentre per comandare gli interruttori utilizzo dei classici shelly collegati tramite wifi, però so che esistono dei relè sonoff che utilizzano zigbee (anche se non li ho mai testati).
Un consiglio che posso darti è di informarti bene prima su quale integrazione di HomeAssistant utilizzare, per evitare spiacevoli sorprese (es. incompatibilità).

Ciao!

**Reply by truzzman** [reply]
Ok, si uso anche degli shelly, ma ripeto la copertura WiFi è quella che è e speravo con zigbee mesh di migliorare la connessione

**Reply by snakuzzo** [reply]
...e comunque, prima di cambiare tutto, prova a testare la copertura della rete con dei device che fungano anche da router a basso prezzo (sostanzialmente tutti quelli alimentati NON a batteria).

**Reply by snakuzzo** [reply]
...considera poi che anche zigbee va sulla 2.4GHz, per cui potresti avere gli stessi problemi. Ovvio che finché non provi con mano non puoi sapere, però anziché cambiare tutto l'ambaradam io proverei prima con qualche extender wifi

**Comment by NdK73** [comment]
Probabilmente, per aggirare il problema dei muri spessi sarebbero più adatti device Z-Wave a frequenza più bassa (433 o 868MHz).

Con ZigBee, per fare bene un sistema mesh, dovresti piazzare dei nodi alimentati (possono anche essere semplici lampade, ovviamente connesse in modo da essere sempre "accese" e pilotate via ZB) che facciano da router in posizioni strategiche.

Come coordinator io ho preso uno ZigStar Lan Gateway (pare che sia in arrivo anche la versione PoE!) che funziona sia con ZHA (semplicissimo) che con Z2M (che **potrebbe** permettere di avere più coordinator che fanno capo ad un'unica istanza di HA, utile se si devono gestire aree geograficamente "lontane" per le quali non è comodo/possibile sfruttare la rete mesh). L'utilità di averlo collegato via LAN è elevata: intanto puoi installarlo dove è più utile, senza essere legato ad un cavo USB, e poi se usi una VM per HA puoi spostarla da un server a un altro senza interruzioni.

**Comment by 50494554524F** [comment]
spenderesti meno con la powerline wifi... (se passa poco ma passa puoi provare anche con dei repeater, costano ancora meno)
